TTHAnalysis/python/plotter/susy-sos-v2-clean/mca-includes/2018/3l_semiddbkg/mca-data.txt

```diff
@@ -1,9 +0,0 @@
-### 2018
```

Comment:
Should we just delete this file? I didn't notice doing the same for the other years, so I guess it was removed by mistake?